lib/dal/src/management/prototype.rs

```diff
@@ -108,6 +119,31 @@ pub struct ManagedComponent {
     geometry: HashMap<String, ManagementGeometry>,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct ManagerComponent {
+    #[serde(flatten)]
```

Comment:
#[serde(flatten)] is an easy way to handle "extending" a serialized type without having to rewrite all the fields.